start.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+DIR="$(cd -P "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Create infrastructure. This also changes the python IP in the node app so that they can communicate
+# The -auto-approve flag makes it not to ask for plan approval. Use at your own risk!
+echo "##### Applying terraform infrastructure #####"
+terraform apply -auto-approve $DIR
```

Comment:
Esto supuestamente corre los archivos de configuración que encuentra en `$DIR`. Eso parecería funcionar, pero por alguna razón, al correrlo desde algún lado que no es la carpeta raíz del proyecto, no me encuentra los providers:
```
➜  Documents terraform apply tp-arq-aws
Plugin reinitialization required. Please run "terraform init".
Reason: Could not satisfy plugin requirements.

Plugins are external binaries that Terraform uses to access and manipulate
resources. The configuration provided requires plugins which can't be located,
don't satisfy the version constraints, or are otherwise incompatible.

2 error(s) occurred:

* provider.aws: no suitable version installed
  version requirements: "(any version)"
  versions installed: none
* provider.template: no suitable version installed
  version requirements: "(any version)"
  versions installed: none

Terraform automatically discovers provider requirements from your
configuration, including providers used in child modules. To see the
requirements and constraints from each module, run "terraform providers".

Error: error satisfying plugin requirements
```

O sea, ahora está todo con absolute paths salvo que el `terraform apply` no estaría funcionando desde otro lado que no sea el root del proyecto